configurations/aws_config.yaml

```diff
@@ -9,3 +9,4 @@ nwp:
 pv_metadata_path: s3://pvsite-ml-models-development/meta_inferred.csv
 
 pv_db_url: ${OCF_PV_DB_URL}
+pvsite_db_url: ${OCF_PVSITE_DB_URL}
```

Comment:
Aren't we writing in the same database as we are reading? Should those two URLs be merged into one?